### DISCUSSION TITLE
Following the reboot of the querynode, are there any approaches to accelerate the data loading procedure?

### DISCUSSION THREAD

**Discussion by UserWangDong** [milvus-io/milvus#28974] [Q&A and General discussion]
Upon the querynode's restart, when confronted with a substantial amount of collection data, it often necessitates a significant amount of time to reload the data. Are there any relevant parameters or methods available to expedite the data loading process on the querynode, thereby facilitating a prompt recovery and restoration of cluster functionality?

**Comment by yhmo** [comment]
It mainly depends on the bandwidth of s3 to read the data files into querynode's memory. The parallel tasks to read the files are internally managed.
